Gemfile.jruby-3.1.lock.release

```diff
@@ -895,111 +896,112 @@ DEPENDENCIES
   gems (~> 1)
   jar-dependencies (= 0.4.1)
   json-schema (~> 2)
-  logstash-codec-avro
-  logstash-codec-cef
-  logstash-codec-collectd
-  logstash-codec-dots
-  logstash-codec-edn
-  logstash-codec-edn_lines
-  logstash-codec-es_bulk
-  logstash-codec-fluent
-  logstash-codec-graphite
-  logstash-codec-json
-  logstash-codec-json_lines
-  logstash-codec-line
-  logstash-codec-msgpack
-  logstash-codec-multiline
-  logstash-codec-netflow
-  logstash-codec-plain
-  logstash-codec-rubydebug
+  logstash-codec-avro (~> 3.4)
+  logstash-codec-cef (~> 6.2)
```

Comment:
we don't need this pinning down any more, we should remove it from the lock file and from Gemfile.template

Reply:
done with commit d6a047cf87b7cadb8b9b41fe814e4c01e0fedda3